package.json

```diff
@@ -28,8 +28,8 @@
     "docker-build": "yarn tsc && yarn workspace example-backend build && yarn workspace example-backend build-image",
     "fix": "backstage-cli repo fix --publish",
     "postinstall": "husky || true",
-    "lint": "backstage-cli repo lint --since origin/master",
-    "lint:all": "backstage-cli repo lint",
+    "lint": "backstage-cli repo lint --since origin/master --quiet",
+    "lint:all": "backstage-cli repo lint --quiet",
```

Comment:
Do we actually want this change? This would be for this repo only and not something that gets shipped. I'm not sure we want to change what has been working well for the project itself.

Reply:
I'm also interested to know if this is something that's wanted. I can remove this, as there isn't really any change in how this project reports things.

Reply:
Removed the `--quiet` portion from `lint` scripts.